project/graph_utils.py

```diff
@@ -0,0 +1,24 @@
+import cfpq_data
+from collections import namedtuple
+from typing import Tuple
+import networkx
+
+GraphInfo = namedtuple("GraphInfo", ["vertices", "edges", "labels"])
+
+
+def get_graph_info(graph) -> GraphInfo:
+    """Returns namedtuple consisted of number of vertices, edges and labels of given graph"""
+    return GraphInfo(
+        graph.number_of_nodes(), graph.number_of_edges(), cfpq_data.get_labels(graph)
+    )
+
+
+def generate_and_save_two_cycle(
+    num_first: int, num_second: int, edge_labels: Tuple[str, str], filename: str
```

Comment:
Имена параметров весьма неоднозначны. Сделайте однозначными. Например так.
```suggestion
    first_cycle_nodes_num: int, second_cycle_nodes_num: int, edge_labels: Tuple[str, str], filename: str
```